update Brother integration examples with new template sensor format

cturra opened the pull request:
## Proposed change
template sensor configurations were changed in 2021.4 and the Brother printer integration documents still reference the legacy format.  just a quick update to correct that.



## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: N/A
- Link to parent pull request in the Brands repository: N/A
- This PR fixes or closes issue: N/A

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

cturra committed:
update Brother integration examples with new template sensor format

template sensor configurations were changed in 2021.4 and the
Brother printer integration documents still reference the legacy
format. just a quick update to correct that.

bieniu commented on source/_integrations/brother.markdown:
```diff
@@ -37,21 +37,23 @@ For some Brother devices, `SNMPv3 read-write access and v1/v2c read-only access`
+    - name: 'Later Printer Paper Jam'
+      unique_id: laser_printer_paper_jam
+      state: >
+        {{ is_state('sensor.<DEVICE_NAME>_status', 'paper jam') }}
```
I'm not sure if we should use `<DEVICE_NAME>` here, user may have changed entity_id. IMO it's better to use `sensor.hl_l2340d_status` as an example.

cturra replied:
agreed.

cturra committed:
replacing DEVICE_ID with hl_l2340d for a real-world example

frenck commented on source/_integrations/brother.markdown:
```diff
@@ -37,21 +37,23 @@ For some Brother devices, `SNMPv3 read-write access and v1/v2c read-only access`
+    - name: 'Later Printer Paper Jam'
+      unique_id: laser_printer_paper_jam
+      state: >
+        {{ is_state('sensor.hl_l2340d_status', 'paper jam') }}
```
I would not recommend setting the `unique_id` in the example. For different reasons:

- It is optional.
- The suggested IDs I would not recommend (use of UUIDs is generally safer).

```suggestion
template:
  - binary_sensor:
    - name: 'Laser Printer Out Of Paper'
      state: >
        {{ is_state('sensor.hl_l2340d_status', 'no paper') }}

  - binary_sensor:
    - name: 'Later Printer Paper Jam'
      state: >
        {{ is_state('sensor.hl_l2340d_status', 'paper jam') }}
```

cturra committed:
Update source/_integrations/brother.markdown

thanks for sharing that information. (i will have some updates to my own home assistant config on this front now too) :)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

frenck reviewed (approved):
Thanks, @cturra 👍 

Happy Hacktoberfest 🎉 

../Frenck
